### PR TITLE
change cloudcore loginfo level

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func (dc *DownstreamController) syncPod(stop chan struct{}) {
 				klog.Infof("send message successfully, operation: %s, resource: %s", msg.GetOperation(), msg.GetResource())
 			}
 		case <-stop:
-			klog.Info("stop syncPod")
+			klog.Error("stop syncPod")
```
Comment (kevin-wangzefeng) [inline cloud/pkg/edgecontroller/controller/downstream.go]: Should not make the log level error since SIG_TERM received will enter this case.

Comment (kadisi) [reply cloud/pkg/edgecontroller/controller/downstream.go]: done

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 2]
```diff
@@ -131,7 +131,7 @@ func (dc *DownstreamController) syncConfigMap(stop chan struct{}) {
 				}
 			}
 		case <-stop:
-			klog.Info("stop syncConfigMap")
+			klog.Warning("Stop eddgecontroller downstream sync configmap loop")
```
Comment (rohitsardesai83) [inline cloud/pkg/edgecontroller/controller/downstream.go]: eddgecontroller -> edgecontroller

Comment (kadisi) [reply cloud/pkg/edgecontroller/controller/downstream.go]: done

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 3]
```diff
@@ -267,7 +267,7 @@ func (dc *DownstreamController) syncEdgeNodes(stop chan struct{}) {
 				break
 			}
 		case <-stop:
-			klog.Info("Stop syncNodes")
+			klog.Warning("Stop edgecontroller downstream sync nodes loop")
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/downstream.go]: nit: s/sync nodes/syncEdgeNodes

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 4]
```diff
@@ -323,7 +323,7 @@ func (dc *DownstreamController) syncService(stop chan struct{}) {
 				return true
 			})
 		case <-stop:
-			klog.Info("Stop sync services")
+			klog.Warning("Stop edgecontroller downstream sync services loop")
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/downstream.go]: nit: s/sync services/syncService

File: cloud/pkg/edgecontroller/controller/downstream.go [hunk 5]
```diff
@@ -418,7 +418,7 @@ func (dc *DownstreamController) syncEndpoints(stop chan struct{}) {
 				})
 			}
 		case <-stop:
-			klog.Info("Stop sync endpoints")
+			klog.Warning("Stop edgecontroller downstream sync endpoints loop")
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/downstream.go]: s/ sync endpoints/syncEndpoints

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 6]
```diff
@@ -328,11 +328,11 @@ func (uc *UpstreamController) updatePodStatus(stop chan struct{}) {
 				}
 
 			default:
-				klog.Infof("pod status operation: %s unsupported", msg.GetOperation())
+				klog.Warningf("pod status operation: %s unsupported", msg.GetOperation())
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/upstream.go]: please start log with caps letters

Comment (sids-b) [reply cloud/pkg/edgecontroller/controller/upstream.go]: applicable to similar places

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 7]
```diff
@@ -507,13 +507,13 @@ func (uc *UpstreamController) queryConfigMap(stop chan struct{}) {
 					klog.Warningf("message: %s process failure, response failed with error: %s", msg.GetID(), err)
 					continue
 				}
-				klog.Warningf("message: %s process successfully", msg.GetID())
+				klog.V(4).Infof("message: %s process successfully", msg.GetID())
 			default:
-				klog.Infof("message: %s process failure, config map operation: %s unsupported", msg.GetID(), msg.GetOperation())
+				klog.Warningf("message: %s process failure, config map operation: %s unsupported", msg.GetID(), msg.GetOperation())
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/upstream.go]: s/config map/configMap

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 8]
```diff
@@ -794,13 +794,13 @@ func (uc *UpstreamController) queryPersistentVolumeClaim(stop chan struct{}) {
...
+				klog.V(4).Infof("message: %s process successfully", msg.GetID())
 			default:
-				klog.Infof("message: %s process failure, persistentvolumeclaim operation: %s unsupported", msg.GetID(), msg.GetOperation())
+				klog.Warningf("message: %s process failure, persistentvolumeclaim operation: %s unsupported", msg.GetID(), msg.GetOperation())
 			}
-			klog.Infof("message: %s process successfully", msg.GetID())
+			klog.V(4).Infof("message: %s process successfully", msg.GetID())
 		case <-stop:
-			klog.Infof("stop queryPersistentVolumeClaim")
+			klog.Error("stop queryPersistentVolumeClaim")
```
Comment (sids-b) [inline cloud/pkg/edgecontroller/controller/upstream.go]: Lets have a warning log for the stop case.

Applicable to similar places

**Comment by sids-b** [comment]
Thanks @kadisi 

/approve

**Pull request merged**